public/app/plugins/panel/heatmap-new/HeatmapPanel.tsx

```diff
@@ -126,7 +126,10 @@ export const HeatmapPanel: React.FC<HeatmapPanelProps> = ({
       getTimeRange: () => timeRangeRef.current,
       palette,
       cellGap: options.cellGap,
-      hideThreshold: options.filterValues?.min, // eventually a better range
+      hideLE: options.filterValues?.min,
+      hideGE: options.filterValues?.max,
```

Comment:
@ryantxu does `filterValues` min/max seem backwards to you?

maybe `hideValues.le` and `hideValues.ge`?

or `showValues`? though we might use this one for actually rendering text in the cells in the future :roll_eyes:

Reply:
🤔 maybe:

```
hideCells: {
  le?: number;
  ge?: number;
},
```
buckets, bins, and cells 😱 

"Values" is the value inside